cypress/support/commands.js

```diff
@@ -29,13 +25,69 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 addMatchImageSnapshotCommand();
 
 Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+  cy.get(`[placeholder="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/users', {
-    email,
-    username,
+Cypress.Commands.add('register', (
+  email = 'riot@qa.team',
+  username = 'riot',
+  password = '12345Qwert!'
+) => {
+  return cy.request('POST', 'https://conduit.mate.academy/api/users', {
+    user: {
+      email,
+      password,
+      username
+    }
+  }).then((response) => ({
+    ...response.body.user,
     password
+  }));
+});
+
+Cypress.Commands.add('login', (user) => {
+  cy.request('POST', 'https://conduit.mate.academy/api/users/login', {
```

Comment:
Ensure that the API endpoint URL for user login is correct and accessible. If the base URL changes, update this endpoint accordingly.